utils/wxrc/GenerateBigXRCFile.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e -o pipefail
+
+# Usage example:
+#
+# The following command line:
+#   './GenerateBigXRCFile.sh' 2000 >'MyInputFileReferencingManyResources.xrc'
+# will generate an XRC file referencing 2000 small PNG images.
+#
+# Compiling this XRC file using `wxrc` and further compiling the generated C++ source code
+# can help us tune our code and avoid slow C++ compilation process
+# (when thousands of input resource files have been given as input).
+
+GenerateExampleImages ()
+{
+    local folder_images='ExampleImages'
+    mkdir -p "${folder_images}/"
+
+    local i
+    local n="${1:-10000}"
+
+    printf '<?xml version="1.0" encoding="UTF-8"?>\n'
+    printf '<resource xmlns="http://www.wxwidgets.org/wxxrc" version="2.5.3.0">\n'
+    for ((i = 0; i < n; ++i)); do
+        local pathname_image; printf -v pathname_image "%s/Example_%04Xh.png" "${folder_images}" "$((i))"
+
+        if ((1)); then
```

Comment:
Sorry, what's the purpose of this line?

Reply:
Thank you. I have removed it (and the corresponding `fi` of the `if`).